### PR TITLE
add the mpassit and upp tasks

### PR DESCRIPTION
**Pull request by guoqing-noaa**

### CODE REVIEW

File: scripts/exrrfs_mpassit.sh
Comment (MatthewPyle-NOAA) [inline scripts/exrrfs_mpassit.sh:1]: Two small comments about comments.  

For the comment at line 16, the UPP will run alongside the model (so as the model completes writing an output file, the UPP will process it within the DATAROOT region.  

The comment on line 35 is misleading - not running the MPAS model in this script.

File: scripts/exrrfs_upp.sh
Comment (MatthewPyle-NOAA) [inline scripts/exrrfs_upp.sh:1]: Same misleading comment about this script running the MPAS model

**Review by MatthewPyle-NOAA — CHANGES_REQUESTED** [review]
Some very minor cleanup to comments, but otherwise looks good to me.

**Comment by guoqing-noaa** [comment]
@MatthewPyle-NOAA  Thanks a lot for the information and review!
I just made changes per the feedback.

**Review by MatthewPyle-NOAA — APPROVED** [review]

**Comment by MatthewPyle-NOAA** [comment]
Will wait a bit before merging in case @hu5970 has comments

**Review by BenjaminBlake-NOAA — APPROVED** [review]
Looks good to me, thanks for addressing Matt's comments.

**Comment by hu5970** [comment]
@guoqing-noaa I have trouble to compile MASSIT on Hera with this branch of the code:
/scratch1/BMC/wrfruc/mhu/rrfs2/v0.0.1/rrfs-workflow/sorc/log.build.mpassit

Do you have any idea on what could be wrong?
Thanks,
Ming

**Comment by guoqing-noaa** [comment]
@hu5970 Thanks for testing this. @EricJames-NOAA made some changes to make MPASSIT work on Hera and I put it in a fork. Let me do a clean test for this and get back to you.

**Pull request merged**